GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/DehydrateTests.cs

```diff
@@ -200,5 +207,24 @@ private ProcessResult RunDehydrateProcess(bool confirm, bool noStatus)
 
             return ProcessHelper.Run(processInfo);
         }
+
+        private string RunHandleProcess(string path)
+        {
+            try
+            {
+                ProcessStartInfo processInfo = new ProcessStartInfo("handle.exe");

```

Comment:
Do all the agents already have handle.exe installed?

Reply:
Yep just put it on all of them.